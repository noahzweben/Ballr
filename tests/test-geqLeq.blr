gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

          if (4 <= 5) {
               print(1);
          }

          if (4 <= 4) {
               print(2); 
          }

          if (4 <= 3) {
               print(3);
          }

          if (5 >= 4) {
               print(4); 
          }

          if (5 >= 6) {
               print(5);
          }

          if (5 >= 5){
               print(6); 
          }

  	}
}