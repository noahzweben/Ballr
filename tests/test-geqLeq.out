1
2
4
6
SDL could not initialize! SDL Error: No available video device
Window initialization failed.